test/tstQueryTreeCallbacks.cpp

```diff
@@ -0,0 +1,343 @@
+/****************************************************************************
+ * Copyright (c) 2012-2020 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_EnableDeviceTypes.hpp" // ARBORX_DEVICE_TYPES
+#include <ArborX_LinearBVH.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+#include <numeric>
+#include <vector>
+
+#include "Search_UnitTestHelpers.hpp"
+
+BOOST_AUTO_TEST_SUITE(Callbacks)
```

Comment:
The name here and other tests should follow file name.

Reply:
Not sure I understand. You want `Callbacks -> QueryTreeCallbacks`?